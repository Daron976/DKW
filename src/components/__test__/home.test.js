import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Home from '../home';

describe('Home component', () => {
  test('home component renders as exepcted', async () => {
    const snapHome = renderer.create(<Home />)
      .toJSON();
    expect(snapHome).toMatchSnapshot();
  });
  test('home component to be present in DOM', async () => {
    render(<Home />);

    const homeComponent = screen.getByTestId('home');

    expect(homeComponent).toBeInTheDocument; //eslint-disable-line
  });
});
