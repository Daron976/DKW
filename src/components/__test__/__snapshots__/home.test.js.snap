// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component home component renders as exepcted 1`] = `
<section
  className="home-content"
  data-testid="home"
>
  <div
    className="home-header"
  >
    <h1>
      Welcome to our webpage
    </h1>
  </div>
  <div
    className="text-container"
  >
    <p
      className="text-content"
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
    </p>
    <p
      className="text-content"
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
    </p>
  </div>
</section>
`;
